hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/DataChecksum.java

```diff
@@ -78,6 +87,18 @@ public static Checksum newCrc32() {
     return new CRC32();
   }
 
+  public static Checksum newCrc32C() {
```

Comment:
again, private. Comment should say "use a volatile to avoid a lock here; re-entrancy unlikely  except in failure mode (and inexpensive)

Reply:
Made it package visible to keep it accessible from Crc32PerformanceTest. Note, newCrc32() is still public. Added javadoc to the method.